src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py

```diff
@@ -206,3 +224,24 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.command('list', 'acr_network_rule_list')
         g.command('add', 'acr_network_rule_add')
         g.command('remove', 'acr_network_rule_remove')
+    
+    with self.command_group('acr scope-map', acr_scope_map_util) as g:
+        g.command('create', 'acr_scope_map_create')
+        g.command('delete', 'acr_scope_map_delete')
+        g.command('update', 'acr_scope_map_update')
+        g.command('show', 'acr_scope_map_show')
+        g.command('list', 'acr_scope_map_list')
+    
+    with self.command_group('acr token', acr_token_util) as g:
+        g.command('create', 'acr_token_create')
+        g.command('delete', 'acr_token_delete')
+        g.command('update', 'acr_token_update')
+        g.command('show', 'acr_token_show')
+        g.command('list', 'acr_token_list')
+
+    with self.command_group('acr token credential', acr_token_credential_generate_util) as g:
+        g.command('generate', 'acr_token_credential_generate')
+
+    with self.command_group('acr token credential', acr_token_util) as g:
+        g.command('add-certificate', 'acr_token_credential_add_certificate')
+        g.command('delete', 'acr_token_credential_delete')
```

Comment:
You can combine the two, and pass in a different util for `generate`?

Reply:
You say, put `g.command('credential add-certificate', 'acr_token_credential_add_certificate')` under `with self.command_group('acr token', acr_token_util) as g` ?

Reply:
Keeping them as they are by now, as there might be design changes.